superset-frontend/src/views/components/Menu.tsx

```diff
@@ -185,6 +185,11 @@ export function Menu({
   const screens = useBreakpoint();
   const uiConig = useUiConfig();
 
+  const getUploadMenulinks: MenuObjectProps[] =
+    menu &&
+    (menu?.filter(item => item.name === 'Upload Data')[0]
```

Comment:
can you use menu?.find() here? That should return the first item it finds.

Reply:
Actually, now I'm curious, where does this value come from? I don't see it on the page anywhere. :)